pkg/k8sapi/k8sutils.go

```diff
@@ -68,6 +70,7 @@ func CreateCachedKubeClient(rawK8SClient client.Client, mapper meta.RESTMapper)
 	vpcCniScheme := runtime.NewScheme()
 	clientgoscheme.AddToScheme(vpcCniScheme)
 	eniconfigscheme.AddToScheme(vpcCniScheme)
+	rcscheme.AddToScheme(vpcCniScheme)
```

Comment:
Similar to https://github.com/aws/amazon-vpc-cni-k8s/pull/2439 , do we need to scope down caching for this CRD by adding an object selector for this node only? Alternatively, we may not want to cache this object since there will always only be one per node, and we only do patch operations

Reply:
Do you mean always use raw client for this operation?